L1Trigger/L1TCaloLayer1/plugins/L1TCaloLayer1.cc

```diff
@@ -81,9 +81,13 @@ class L1TCaloLayer1 : public edm::EDProducer {
   edm::EDGetTokenT<HcalTrigPrimDigiCollection> hcalTPSource;
   std::string hcalTPSourceLabel;
   
-  std::vector< std::vector< std::vector < uint32_t > > > ecalLUT;
-  std::vector< std::vector< std::vector < uint32_t > > > hcalLUT;
-  std::vector< std::vector< uint32_t > > hfLUT;
+  std::vector< std::vector< std::vector< std::vector < uint32_t > > > > ecalLUT;
```

Comment:
There are CaloLayer1 Tower calibrations in bins of pt, eta, and now also of phi.
Will add comment.